ANMN/temperature_grid_product/anmn_temp_gridded_product.py

```diff
@@ -513,9 +513,14 @@ def main(incoming_file_path, deployment_code, output_dir, plot_comparaison=False
     s3_bucket_prefix = 'https://s3-ap-southeast-2.amazonaws.com/imos-data'
     logging           = IMOSLogging()
     logger            = logging.logging_start(os.path.join(os.environ['WIP_DIR'], 'anmn_temp_grid.log'))
-    list_fv01_url     = wfs_request_matching_file_pattern('anmn_all_map', '%%Temperature/%%_TZ_%%_FV01_%s%%' % deployment_code, s3_bucket_url=True)
-    list_fv01_url.extend(wfs_request_matching_file_pattern('anmn_all_map', '%%Temperature/%%_TE_%%_FV01_%s%%' % deployment_code, s3_bucket_url=True))
-    list_fv01_url.extend(wfs_request_matching_file_pattern('anmn_all_map', '%%Temperature/%%_TPE_%%_FV01_%s%%' % deployment_code, s3_bucket_url=True))
+    list_fv01_url     = wfs_request_matching_file_pattern('anmn_all_map', '%%_%%T%%_%%_FV01_%s%%' % deployment_code, s3_bucket_url=True)
```

Comment:
I think that pattern needs to be a bit more specific. In particular `'%%_%%T%%_%%_'` will match all files because there's a 'T' in the timestamp.
Not sure what is possible with the patterns used by WFS. With regex you could do something like 
```regex
IMOS_ANMN-[A-Z]+_[^VW_]*T[^VW_]*_.*_FV01
```